Slam/Queue/QueueManagerProtocol.swift

```diff
@@ -0,0 +1,6 @@
+public protocol QueueManager {
+    var queueView: QueueView { get }
+
+    func loadQueue()
```

Comment:
We should probably disable the buttons until the initial queue is loaded.

If that's the case, then `viewDidLoad` and `clickRefreshQueue` should probably call the same function.

I would probably get rid of `loadQueue` and rename `reloadQueue` to `loadQueue`.